daml-script/dump/src/main/scala/com/daml/script/dump/Encode.scala

```diff
@@ -285,6 +287,27 @@ private[dump] object Encode {
     ((bind + submit :+ " [] do") / body).hang(2)
   }
 
+  private[dump] def encodeACS(
+      partyMap: Map[Party, String],
+      cidMap: Map[ContractId, String],
+      cidRefs: Set[ContractId],
+      acs: Seq[CreatedEvent],
+      batchSize: Int,
+  ): Doc = {
+    val acsSize = acs.length
+    val numBatches: Int = if (acsSize % batchSize == 0) {
```

Comment:
can we use [grouped](https://www.scala-lang.org/api/current/scala/collection/Iterator.html#grouped[B%3E:A](size:Int):Iterator.this.GroupedIterator[B]) here?

Reply:
Thanks, I was looking for this but somehow missed it.